apps/test/unit/templates/teacherNavigation/LessonMaterialsContainerTest.tsx

```diff
@@ -55,7 +55,7 @@ describe('LessonMaterialsContainer', () => {
   });
 
   it('renders the component and dropdown with lessons', () => {
```

Comment:
Can we add a test that the `NoUnitAssignedView` is shown and not the LessonMaterials stuff